### PR TITLE
Merge fee cache and fix sample conf

### PR DESCRIPTION
**Pull request by yyforyongyu**
Merge #8484 to master and add the missing `[fee]` section.

### CODE REVIEW

**Review by guggero — APPROVED** [review]
LGTM, thanks! Can you rebase to get rid of the merge commit though?

**Comment by yyforyongyu** [comment]
> LGTM, thanks! Can you rebase to get rid of the merge commit though?

Done!

**Review by Roasbeef — APPROVED** [review]
LGTM 🐊

File: peer/brontide.go [hunk 1]
```diff
@@ -1146,8 +1146,8 @@ func (p *Brontide) addLink(chanPoint *wire.OutPoint,
 		),
 		BatchSize:               p.cfg.ChannelCommitBatchSize,
 		UnsafeReplay:            p.cfg.UnsafeReplay,
-		MinFeeUpdateTimeout:     htlcswitch.DefaultMinLinkFeeUpdateTimeout,
-		MaxFeeUpdateTimeout:     htlcswitch.DefaultMaxLinkFeeUpdateTimeout,
+		MinUpdateTimeout:        htlcswitch.DefaultMinLinkFeeUpdateTimeout,
```
Comment (yyforyongyu) [inline peer/brontide.go:1149]: a second look at this change - these are unrelated to the feeurl timeout, and giving them the same names can be a bit confusing, will change.
